.github/workflows/docker_buildx-on-tag-publish.yml

```
@@ -0,0 +1,56 @@
+            exit 1
+          }
+
+      - name: Set outputs for downstream jobs
+        run: |
+          echo "APP_NAME=$APP_NAME" >> $GITHUB_OUTPUT
+          echo "VERSION=$VERSION" >> $GITHUB_OUTPUT
+
+  deploy-docker-buildx-image:
+    needs: tag-checker-job
+    name: Run plz action for Docker BuildX
+    uses: ./.github/workflows/run-plz.yml@main
```

actionlint (1.7.7)

reusable workflow call "./.github/workflows/run-plz.yml@main" at "uses" is not following the format "owner/repo/path/to/workflow.yml@ref" nor "./path/to/workflow.yml". see https://docs.github.com/en/actions/learn-github-actions/reusing-workflows for more details

(workflow-call)